### PR TITLE
Add static checks in CI

### LINKED ISSUE #359: Add go code style checks for cloud code in CI
**What would you like to be added**:
Static checks like gofmt, golint, govet

**Why is this needed**:
To improve code quality.

### PR DESCRIPTION
**Pull request by u5surf**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
I added code static check for cloud code in travis-ci

**Which issue(s) this **
Fixes #359

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,3 +34,5 @@ jobs:
       name: "cross build edge"     # names the fifth Tests stage job
     - script: make e2e_test
       name: "e2e_test"     # names the seventh Tests stage job
+    - script: make cloud_lint
+      name: "cloud_lint"     # names the second Tests stage job
```
Comment (sids-b) [inline .travis.yml]: nit; names the second -> names the eighth

**Comment by u5surf** [comment]
@sids-b
I just fixed comment in .travis.yml, but e2e test was failed. I cannot understand its fault.

**Comment by m1093782566** [comment]
/retest

cc @pavan187 to take a look at the failing e2e tests.

```
Summarizing 2 Failures:
[Fail] Application deployment test in E2E scenario Test application deployment and delete deployment using deployment spec [AfterEach] E2E_APP_DEPLOYMENT_1: Create deployment and check the pods are coming up correctly 
/home/travis/gopath/src/github.com/kubeedge/kubeedge/tests/e2e/utils/pod.go:138
[Fail] Application deployment test in E2E scenario Test application deployment and delete deployment using deployment spec [AfterEach] E2E_APP_DEPLOYMENT_2: Create deployment with replicas and check the pods are coming up correctly 
/home/travis/gopath/src/github.com/kubeedge/kubeedge/tests/e2e/utils/pod.go:138
```

**Comment by sids-b** [comment]
Once @pavan187 captures the log of CI failure, we can rerun the CI and merge :) Thanks @u5surf

**Pull request merged**